### ISSUE TITLE
Fallback choices become visible after reloading the story state

### ISSUE THREAD

**Issue by arashikou** [y-lohse/inkjs#912]
## Describe the bug

Fallback choices are supposed to be invisible to the player, taken automatically by Ink if no other choices remain. However, in InkJS, they can become visible if you save the story state to JSON while at a choice block with a fallback and then restore from the JSON. The now-visible fallback can even be selected and followed.

## To Reproduce

I have [a gist](https://gist.github.com/arashikou/c2e6f645c50fada635f10641415aca99) that demonstrates the issue using a looping failure block similar to the "find_help" example in [_Writing With Ink_'s section on fallback choices](https://github.com/inkle/ink/blob/master/Documentation/WritingWithInk.md#fallback-choices). (Excuse all the Apache 2.0 license stuff; my employer requires it.)

Essentially, if the story state is at a choice block with a fallback and you save the state with `story.state.ToJson()`, then if you later restore that state with `story.state.LoadJson()`, the fallback choice will become visible as a regular choice with no text.

## Expected behavior

From _Writing With Ink_: "Fallback choices are never displayed to the player".

## Ink files

See demo.ink in [this gist](https://gist.github.com/arashikou/c2e6f645c50fada635f10641415aca99), along with main.mjs.

## Environment
 - OS: Windows
 - Node.js version: 16.10.0, but I've also seen this behavior in the browser
 - Web browser version: Firefox 93, but I've also seen this behavior in Node.js
 - _inklecate_ version: 1.0.0
 - _inkjs_ version: 2.0.0

**Comment by y-lohse** [comment]
Yeah that… sounds like a bad bug 😬 thanks for the reproduction material, I won't be able to look at this in the near future but happy to support if anyone wants to tackle this!

**Comment by smwhr** [comment]
Is this still happening ?
